msteams-platform/concepts/build-and-test/teams-developer-portal.md

```diff
@@ -72,7 +72,77 @@ On the **Overview** page, you can see the total number of active users for your
 | *Monthly R30* | The default usage metric. It shows you the count of unique active users that used your app within that rolling 30-day window in UTC. |
 | *Daily* | Shows you the count of unique active users that used your app in a given day in UTC. |
 
-Monthly and daily usage is shown for the past seven, 30 days, and 60 days. You should see usage reflected for a given day within 24-48 hours. Usage for new apps can take up to 3-5 days to display.
+You should see usage reflected for a given day within 24-48 hours. Usage for new apps can take up to 3-5 days to display.
+
+Click **View Details** or navigate to the **Analytics** page to learn more about who's using your app and how they're using it to answer queries like:
+
+* How many active users do I have on a monthly, weekly, and daily basis?
+* Are these new users lapsed, revived, or returned users?
+* Do these users use my app more than five times each month?
+* What is the breakdown of active users by platform/operating system?
+
+### Filtering for Aggregation Type and App
+
+At the top of the report, you will see a **Filter** button that allows you to slice and dice all subsequent metrics by the following:
```

Comment:
The Filter button allows you to analyze all the subsequent metrics by the following: